platform/linux-generic/odp_packet_io.c

```diff
@@ -376,8 +376,10 @@ int odp_pktio_close(odp_pktio_t hdl)
 	int res;
 
 	entry = get_pktio_entry(hdl);
-	if (entry == NULL)
+	if (entry == NULL) {
+		ODP_ERR("Bad handle\n");
```

Comment:
The commit log says "add debug prints" so shouldn't this be `ODP_DBG()` rather than `ODP_ERR()`? Same comment for the rest of the `ODP_ERR()` calls here.

Reply:
ODP_ERR() is used throughout the file to report errors. So, ODP_ERR is used also for these errors. The reader of commit log gets the idea of prints been added to error conditions (no functional changes just prints added).

Reply:
OK, but then I'd change the commit log to something like "add error prints to log bad handles". That would make the log and the code consistent.

Reply:
Changed "debug" to "error" in v3.